try.py

```diff
@@ -0,0 +1,107 @@
+# import os
+# import cv2
```

Comment:
what does this script do?

Reply:
This is just to try things separately. I can remove this from the repo as well.

Reply:
OK please remove unnecessary ones.